.github/workflows/release-nethermind.yml

```diff
@@ -14,54 +14,47 @@ permissions:
 
 jobs:
   build-nethermind:
-    name: Building Nethermind Packages
+    name: Build Nethermind packages
     runs-on: ubuntu-latest
     env:      
       RELEASE_DIRECTORY: /home/runner/work/nethermind/nethermind
       LIN_RELEASE: nethermind-lin-x64
-      OSX_RELEASE: nethermind-osx-x64
-      WIN_RELEASE: nethermind-win-x64
       LIN_ARM64_RELEASE: nethermind-lin-arm64
+      WIN_RELEASE: nethermind-win-x64
+      OSX_RELEASE: nethermind-osx-x64
       OSX_ARM64_RELEASE: nethermind-osx-arm64
-      LINUX: linux-x64
```

Comment:
Why are some variables removed here? Some of them are used in scripts, please add them back (or change the scripts) 

One example (but there are probably more) 

build-runner.sh uses 
- $LINUX
- $LINUX_ARM64
- $OSX
- $OSX_ARM64
- $WIN10

Reply:
Good catch

Reply:
Looks like we have some inconsistency with these env vars across our scripts. Gonna dm you.